core/eolearn/core/utils/testing.py

```diff
@@ -0,0 +1,83 @@
+"""
+The eodata module provides core objects for handling remote sensing multi-temporal data (such as satellite imagery).
+
+Credits:
+Copyright (c) 2017-2022 Matej Aleksandrov, Matej Batič, Grega Milčinski, Domagoj Korais, Matic Lubej (Sinergise)
+Copyright (c) 2017-2022 Žiga Lukšič, Devis Peressutti, Tomislav Slijepčević, Nejc Vesel, Jovan Višnjić (Sinergise)
+Copyright (c) 2017-2022 Anže Zupanc (Sinergise)
+Copyright (c) 2019-2020 Jernej Puc, Lojze Žust (Sinergise)
+Copyright (c) 2017-2019 Blaž Sovdat, Andrej Burja (Sinergise)
+
+This source code is licensed under the MIT license found in the LICENSE
+file in the root directory of this source tree.
+"""
+import datetime as dt
+from dataclasses import dataclass, field
+from typing import List, Optional, Tuple
+
+import numpy as np
+import pandas as pd
+
+from sentinelhub import CRS, BBox
+
+from ..constants import FeatureType
+from ..eodata import EOPatch
+from ..utils.parsing import FeatureParser
+
+DEFAULT_BBOX = BBox((0, 0, 100, 100), crs=CRS("EPSG:32633"))
+
+
+@dataclass
+class PatchGeneratorConfig:
+    """Dataclass containing a more complex setup of the PatchGenerator class."""
+
+    num_timestamps: int = 5
+    timestamps_range: Tuple[dt.datetime, dt.datetime] = (dt.datetime(2019, 1, 1), dt.datetime(2019, 12, 31))
+    timestamps: List[dt.datetime] = field(init=False, repr=False)
+
+    num_random_features: int = 5
```

Comment:
this is no longer relevant right?

Reply:
right, removed. good catch!

Reply:
damn, I thought I pushed... Seems like I didn't. @zigaLuksic do I edit it straight on `develop`? Or you can remove it one of the incoming MRs...

Reply:
develop is/should be protected.
you can do a tiny MR for that and merge it yourself :)